src/client/eppo-client.ts

```diff
@@ -175,8 +187,8 @@ export default class EppoClient implements IEppoClient {
     flagKey: string,
     subjectAttributes: Record<string, EppoValue> = {},
     assignmentHooks: IAssignmentHooks | undefined,
-    valueType: ValueType,
-  ): EppoValue | null {
```

Comment:
It was confusing to me to have a null state in `EppoValue` and also to be returning nulls, so I decided on this pattern: return `EppoValue.Null()` if internal to the SDK logic and only return `null` if we're at returning the outermost get<Typed?>Assignment call